scripts/build/change-packages.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
is this file used? cant find the usage in this PR, I also don't quite get why this is needed - especially when you have removed those devDeps etc in this PR

or is it a helper script which u have used to modify packages in this PR + it can be used to modify them in the future? if so it would be great to add a comment about this

Reply:
It's a helper script that can be used to modify the packages in the future, I'll add a comment.